Makefile

```diff
@@ -17,6 +17,8 @@ OBJECTS=$(SOURCES:.c=.o)
 TESTABLE_OBJECTS=$(filter-out main.o,$(SOURCES:.c=.o))
 TARGET=wayuu
 
+slib: CCFLAGS:=$(CCFLAGS) -fpic
```

Comment:
Why is this needed? It's not obvious to me that we need to modify CCFLAGS for this target